difficulties.lua

```diff
@@ -24,6 +43,19 @@ function HandleDifficultyCommand ( Split, Player )
 	if (Split[2] == "peaceful") or (Split[2] == "0") or (Split[2] == "p") then
 		SetWorldDifficulty(Player:GetWorld(), 0)
 		SendMessage( Player, "World difficulty set to peaceful" )
+		
+		--Remove mobs which are not allowed in peaceful
+		Player:GetWorld():ForEachEntity(
+			function (a_Entity)
+				if not(a_Entity:IsMob()) then
+					return
+				end
+				local Monster = tolua.cast(a_Entity, "cMonster")
+				if (Monster:GetMobFamily() == cMonster.mfHostile) then

```

Comment:
Could you try if this works without using `tolua.cast`? I believe the bindings should work in an intelligent way - if a `cEntity` class is sent to Lua, it should actually be the correct class already, so this should work directly:
```lua
-- (Keep the IsMob() check)
if (a_Entity:GetMobFamily() == cMonster.mfHostile) then
```